send-trace-to-jaeger.mjs

```diff
@@ -0,0 +1,65 @@
+import fs from 'fs'
```

Comment:
Should we move this to the scripts folder so it's not in the root of the repo?

Reply:
Done!